README.md

```diff
@@ -91,10 +104,9 @@ For those developers who want to or need to work on the hydra-head gem
 itself, see the [Instructions for
 Contributors](http://github.com/samvera/hydra-head/wiki/For-Contributors)
 
-## Acknowledgments
-
-### Design & Strategic Contributions
+# Acknowledgments
 
-The Hydra Framework would not exist without the extensive design effort undertaken by representatives of repository initiatives from Stanford University, University of Virginia, University of Hull and MediaShelf LLC.  Contributors to that effort include Tom Cramer, Lynn McRae, Martha Sites, Richard Green, Chris Awre, and Matt Zumwalt.
```

Comment:
Is there a place that these acknowledgments have moved to or a place that we know they currently exist?  I'd hate to just disappear them.

Reply:
I didn't move them anywhere. I can leave it, but I assumed it was superseded by the newly recommended piece at the end of the README.